src/wallet/gtest/test_wallet.cpp

```diff
@@ -2200,14 +2200,15 @@ TEST(WalletTests, GenerateUnifiedAddress) {
     EXPECT_EQ(uaResult, expected);
 
     // Create an account, then generate an address for that account.
-    auto skpair = wallet.GenerateNewUnifiedSpendingKey();
-    uaResult = wallet.GenerateUnifiedAddress(skpair.second, {ReceiverType::P2PKH, ReceiverType::Sapling});
+    auto ufvkpair = wallet.GenerateNewUnifiedSpendingKey();
+    auto ufvk = ufvkpair.first;
+    auto account = ufvkpair.second;
+    uaResult = wallet.GenerateUnifiedAddress(account, {ReceiverType::P2PKH, ReceiverType::Sapling});
```

Comment:
Can this test Orchard yet?

Reply:
Not yet; that will be in a successor PR that @str4d is working on.